qa/L0_cmdline_trace/test.sh

```diff
@@ -104,8 +104,8 @@ fi
 
 set -e
 
-# trace-rate == 1, trace-level=MIN make sure every request is traced
```

Comment:
Keep one of the MIN and one of the MAX test cases to cover backwards compatibility